pandas/tests/indexing/test_at.py

```diff
@@ -16,16 +16,6 @@
 import pandas._testing as tm
 
 
-def test_at_timezone():
```

Comment:
This test could be incorporated into `test_at_setitem_expansion`, so I removed it, I hope that it is fine